tests/run/ext_attr_getter.srctree

```diff
@@ -175,7 +179,20 @@ def check_10(getter.Foo f):
     return f.fieldF1 != 10
 
 def vec0(getter.Foo f):
-        return f.vector[0]
+    return f.vector[0]
+
+def check_binop(getter.Foo f):
+    return f.fieldF1 / 10
+
+######## getter2.pyx ########
+
+cimport getter
+
+def check_pyobj(getter.Foo f):
+    return bool(f.fieldF1)
+ 
+def check_unary(getter.Foo f):
```

Comment:
Shouldn't these functions also be called somewhere in order to actually test them?

Reply:
Simple adding them here was enough to trigger compilation issues. It can't hurt to add a call.

Reply:
added a call.